### PR TITLE
Privacy

### PR DESCRIPTION
**Pull request by marcoscaceres**
Address some of the privacy/security issues that came in as part of PING. 

@npdoty, would appreciate your review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,6 +231,30 @@
       </p>
       <section>
         <h3>
+          Security considerations
+        </h3>
+        <p>
+          When <a>installing</a> a web application, it is RECOMMENDED that user
```
Comment (kenchris) [inline index.html]: *the* user agent

File: index.html [hunk 2]
```diff
@@ -231,6 +231,30 @@
...
+          agent allow the user to inspect the icon, name, <a>start URL</a>,
+          origin, etc. pertaining to a web application. This is to give a user
```
Comment (kenchris) [inline index.html]: the user?

File: index.html [hunk 3]
```diff
@@ -231,6 +231,30 @@
...
+          a opportunity to make a conscious decision about approving, and
```
Comment (kenchris) [inline index.html]: an...

File: index.html [hunk 4]
```diff
@@ -231,6 +231,30 @@
...
+          possibly modify, the information pertaining to the web application
+          before installing it. This also gives the user an opportunity to see
```
Comment (kenchris) [inline index.html]: to know?

File: index.html [hunk 5]
```diff
@@ -231,6 +231,30 @@
...
+          example, using an unexpected icon or name.
+        </p>
+        <p>
+          It is RECOMMENDED that user agents prevent other applications from
+          determining which applications are installed on the system (e.g., via
+          a timing attack on the user agent's cache). This could be done by,
+          for example, invalidating from the user agent's cache resources were
+          linked to from the manifest (for example, icons) after a web
+          application is <a>installed</a> - or by using an entirely different
+          cache from that used for regular web browsing.
```
Comment (kenchris) [inline index.html]: than that?

**Comment by marcoscaceres** [comment]
@kenchris, fixed typos! Thank you :beer:!

**Comment by marcoscaceres** [comment]
@npdoty, please file additional feedback as new issues.

**Pull request merged**